### DISCUSSION TITLE
Additional feature - Liked Posts

### DISCUSSION THREAD

**Discussion by raghusaripalli** [raghusaripalli/GatorGiveaway#89] [General]
As the posts grow exponentially, users might want to get back to some posts they like. So a good feature to add would be the liked posts.

Features on UI would be a like button on each post.
A liked posts tab in the top navbar.
Also, a button to, unlike a post.

On the server-side, APIs and DB change to support all the above-mentioned features.

**Comment by tejasri555** [comment]
Great idea! Also, we can have a comments section for the posts. So if users have any reviews or doubts they can post in the comment section.

**Comment by ksmSujay95** [comment]
For liked posts, will add a favorite icon below the item's image on the Post's card which acts as both like and unlike of a post. Also, Fav's list tab in the top nav bar redirects to the favorites list page on click.
